internal/services/oracle/oracle_autonomous_database_cross_region_disaster_recovery_resource.go

```diff
@@ -0,0 +1,336 @@
+
+type AutonomousDatabaseCrossRegionDisasterRecoveryResourceModel struct {
+	Location                     string            `tfschema:"location"`
+	Name                         string            `tfschema:"name"`
+	ResourceGroupName            string            `tfschema:"resource_group_name"`
+	Tags                         map[string]string `tfschema:"tags"`
+	RemoteDisasterRecoveryType   string            `tfschema:"remote_disaster_recovery_type"`
+	SourceAutonomousDatabaseId   string            `tfschema:"source_autonomous_database_id"`
+	AutoScalingEnabled           bool              `tfschema:"auto_scaling_enabled"`
+	AutoScalingForStorageEnabled bool              `tfschema:"auto_scaling_for_storage_enabled"`
+	BackupRetentionPeriodInDays  int64             `tfschema:"backup_retention_period_in_days"`
+	CharacterSet                 string            `tfschema:"character_set"`
+	ComputeCount                 float64           `tfschema:"compute_count"`
+	ComputeModel                 string            `tfschema:"compute_model"`
+	DataStorageSizeInTb          int64             `tfschema:"data_storage_size_in_tb"`
+	DatabaseVersion              string            `tfschema:"database_version"`
+	DatabaseWorkload             string            `tfschema:"database_workload"`
+	DisplayName                  string            `tfschema:"display_name"`
+	LicenseModel                 string            `tfschema:"license_model"`
+	MtlsConnectionRequired       bool              `tfschema:"mtls_connection_required"`
+	NationalCharacterSet         string            `tfschema:"national_character_set"`
+	SubnetId                     string            `tfschema:"subnet_id"`
+	VnetId                       string            `tfschema:"virtual_network_id"`
+
+	// Optional
+	CustomerContacts                 []string `tfschema:"customer_contacts"`
+	ReplicateAutomaticBackupsEnabled bool     `tfschema:"replicate_automatic_backups_enabled"`
+}
+
+func (AutonomousDatabaseCrossRegionDisasterRecoveryResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"location": commonschema.Location(),
+
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validate.AutonomousDatabaseName,
+			ForceNew:     true,
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"display_name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.AutonomousDatabaseName,
+		},
+
+		// Cross Region Disaster Recovery
+		// Required
+
+		"source_autonomous_database_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: autonomousdatabases.ValidateAutonomousDatabaseID,
+		},
+
+		"subnet_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: commonids.ValidateSubnetID,
+		},
+
+		"virtual_network_id": {
```

Comment:
since virtual network id can be got from `subnet_id`, shall we remove `virtual_network_id`?